setup.cfg

```diff
@@ -23,6 +23,7 @@ python_requires = >=3.7
 install_requires =
     dipy>=1.3.0
     scikit-image>=0.14.2
+    nest-asyncio>=1.5.1
```

Comment:
do we really need this patch?

Reply:
If not used, I was getting a `RuntimeError: This event loop is already running` when run in a jupyter notebook where the event loop is already running. This patch allows the event loop to be nested.

Here is a little more details extracted from https://pypi.org/project/nest-asyncio/:

> By design `asyncio` does not allow its event loop to be nested. This presents a practical problem: When in an environment where the event loop is already running it’s impossible to run tasks and wait for the result. Trying to do so will give the error “`RuntimeError: This event loop is already running`”.

Reply:
I made a test with the latest version of my implementation where I am creating a dedicated event loop and it seems it still returning me with the error: `RuntimeError: Cannot run the event loop while another loop is running`